test/viewModels/DepartureStatusSpec.scala

```diff
@@ -60,11 +60,11 @@ class DepartureStatusSpec extends SpecBase with Generators with ScalaCheckProper
     "include tad link on ReleasedForTransit status" in {
       forAll(arbitrary[Departure]) {
         departure =>
-          val dep             = departure.copy(status = "ReleasedForTransit")
+          val dep             = departure.copy(status = "ReleaseForTransit")
```

Comment:
status name, is this the one we want to go with?

Reply:
Yes 👍